docs/getting-started/agentless-integrations.asciidoc

```diff
@@ -3,6 +3,31 @@
 
 Agentless integrations provide a means to ingest data while avoiding the orchestration, management, and maintenance needs associated with standard ingest infrastructure. Using agentless integrations makes manual agent deployment unnecessary, allowing you to focus on your data instead of the agent that collects it. 
 
-We currently support one agentless integration: cloud security posture management (CSPM). Using this integration's agentless deployment option, you can enable Elastic's CSPM capabilities just by providing the necessary credentials. Agentless CSPM deployments support AWS, Azure, and GCP accounts.
+We support the following integrations:
 
-To learn more about agentless CSPM deployments, refer to the getting started guides for CSPM on <<cspm-get-started, AWS>>,  <<cspm-get-started-azure, Azure>>, or <<cspm-get-started-gcp, GCP>>. 
\ No newline at end of file
+[discrete]
+== Generally available (GA) agentless integrations
+Cloud security posture management (CSPM). Using this integration's agentless deployment option, you can enable Elastic's CSPM capabilities just by providing the necessary credentials. Agentless CSPM deployments support AWS, Azure, and GCP accounts.
+
+To learn more about agentless CSPM deployments, refer to the getting started guides for CSPM on <<cspm-get-started, AWS>>,  <<cspm-get-started-azure, Azure>>, or <<cspm-get-started-gcp, GCP>>. 
+
+## Beta agentless integrations
+
+WARNING: Agentless deployment for the following integrations is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
```

Comment:
Nit and optional edit ofc: bullets seem more appropriate for the following list since it's unordered.